.changeset/neat-brooms-allow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/config-loader': patch
+---
+
+Each piece of the configuration schema is now validate upfront, in order to produce more informative errors.
```

Comment:
```suggestion
Each piece of the configuration schema is now validated upfront, in order to produce more informative errors.
```

Reply:
I am waiting for the moment when these random words in changeset file name will mean something.

Like `.changeset/doggo-likes-pets`